spec/lib/importer/csv_importer_spec.rb

```diff
@@ -1,39 +1,23 @@
 require 'rails_helper'
 require 'importer'
+require 'aws-sdk'
```

Comment:
i'm getting an error with this line: ```
An error occurred while loading ./spec/lib/importer/csv_importer_spec.rb.
Failure/Error: require 'aws-sdk'

LoadError:
  cannot load such file -- aws-sdk
# ./spec/lib/importer/csv_importer_spec.rb:3:in <top (required)>'
No examples found.

Randomized with seed 40214


Finished in 0.13636 seconds (files took 8.31 seconds to load)
0 examples, 0 failures, 1 error occurred outside of examples
```

i think we might need to mess with our `Gemfile` groups a bit, and maybe we only want to require the aws-sdk module that we're using in this spec (although i don't care a ton about that right now)

Reply:
Can you try `bundle install --with=aws` and see if that gets past the error?

Reply:
cool, doing that and setting up minio with the updated instructions worked